cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height_test.go

```diff
@@ -24,7 +24,9 @@ func TestReExecuteBlock(t *testing.T) {
 		// bootstrap to init highest executed height
 		bootstrapper := bootstrap.NewBootstrapper(unittest.Logger())
 		genesis := unittest.BlockHeaderFixture()
-		err := bootstrapper.BootstrapExecutionDatabase(db, unittest.StateCommitmentFixture(), genesis)
+		rootSeal := unittest.Seal.Fixture()
+		unittest.Seal.WithBlock(genesis)(rootSeal)
```

Comment:
```suggestion
		rootSeal := unittest.Seal.Fixture(unittest.Seal.WithBlock(genesis))
```
pretty sure it's intended to be used like this